src/app/forms/validators/custom-validators.ts

```diff
@@ -181,7 +181,7 @@ export class CustomValidators {
   }
 
   static email(): ValidatorFn {
-    return this.customPattern(['^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$', 'Enter an email address in the correct format, like name@example.com']);
+    return this.customPattern(['^[\\w\\-\\.]+@([\\w-]+\\.)+[\\w-]{2,}$', 'Enter an email address in the correct format, like name@example.com']);
```

Comment:
has the first `+` been removed for a reason? It stops [plus address email addresses](https://learn.microsoft.com/en-us/exchange/recipients-in-exchange-online/plus-addressing-in-exchange-online) such as `foo+bar@email.com` from being entered if you don't have it in there.

Reply:
Gunna miss this guy

Reply:
only because it wasn't in the validator I'd found - added back in and re-tested locally

Reply:
I also found a email validator that didn't support them... I've found from trying to use them personally that about 25% of websites won't allow it... which is quite annoying.